common/scylla_create_devices

```diff
@@ -47,7 +47,7 @@ def create_raid(devices):
         print(f"{scylla_path} is already mounted. Will not run 'scylla_raid_setup'!")
         sys.exit(0)
     run(["/opt/scylladb/scripts/scylla_raid_setup", "--raiddev", "/dev/md0", "--disks", ",".join(devices),
-         "--root", "/var/lib/scylla", "--volume-role", "all", "--update-fstab"], check=True)
+         "--root", "/var/lib/scylla", "--raid-level", raid_level, "--volume-role", "all", "--update-fstab"], check=True)
```

Comment:
if you call run without `shell=True` you need to pass list of strings
```suggestion
         "--root", "/var/lib/scylla", "--raid-level", str(raid_level), "--volume-role", "all", "--update-fstab"], check=True)
```
otherwise, you will get Exception `TypeError: expected str, bytes or os.PathLike object, not int`:
![image](https://user-images.githubusercontent.com/5269241/146957991-4165477b-f78a-4ee4-af96-261b211c8bcb.png)

Reply:
fixed